docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -565,12 +577,14 @@ Redo the operation of the other role:
  * Description: Corporate owned machines
  * Max nodes per user: 1
 
-Using the Consultant Role
-~~~~~~~~~~~~~~~~~~~~~~~~~
+Let's say we have two roles: employee and corporate_machine (defined above).
 
-Let's say we have two roles: guest and employee. First, we define them *Configuration -> Policies and Access Control -> Roles*.
+Now, we want to authenticate employees and their corporate machines using Active Directory (over LDAP), both using PacketFence's captive portal.
```

Comment:
That's a tiny bit misleading. In fact, it's the role computation for employees and machines that is done over LDAP. Actual authentication is done over EAP-PEAP, so ntlm_auth with the PacketFence server joined to the AD domain.